lib/handlers/ReactNative.js

```diff
@@ -137,7 +137,9 @@ class SendHandler extends Handler
 					'addProducer() | calling pc.setLocalDescription() [offer:%o]',
 					offer);
 
-				return this._pc.setLocalDescription(offer);
+				const offerDesc = new RTCSessionDescription(offer);
```

Comment:
I will move this to the `if` block above since, if simulcast is not used, `offer` is already a `RTCSessionDescription` object.

Reply:
No, it's ok as you did.